phpBB/includes/functions_compatibility.php

```diff
@@ -943,3 +943,76 @@ function parse_cfg_file($filename, $lines = false)
 
 	return $parsed_items;
 }
+
+/**
+* Wraps an url into a simple html page. Used to display attachments in IE.
+* this is a workaround for now; might be moved to template system later
+* direct any complaints to 1 Microsoft Way, Redmond
+*
+* @deprecated: 3.3.0-dev (To be removed: 4.0.0)
+*/
+function wrap_img_in_html($src, $title)
+{
+	echo '<!DOCTYPE html>';
+	echo '<html>';
+	echo '<head>';
+	echo '<meta charset="utf-8">';
+	echo '<meta http-equiv="X-UA-Compatible" content="IE=edge">';
+	echo '<title>' . $title . '</title>';
+	echo '</head>';
+	echo '<body>';
+	echo '<div>';
+	echo '<img src="' . $src . '" alt="' . $title . '" />';
+	echo '</div>';
+	echo '</body>';
+	echo '</html>';
+}
+
+/**
+* Garbage Collection
+*
+* @param bool $exit		Whether to die or not.
+*
+* @return null
+*
+* @deprecated: 3.3.0-dev (To be removed: 4.0.0)
+*/
+function file_gc($exit = true)
```

Comment:
Don't think this is needed anymore either.

Reply:
Yes it is